libs/@guardian/source-react-components/src/button/theme.ts

```diff
@@ -83,6 +86,7 @@ export const themeButton: ThemeButton = {
 	backgroundSecondary: palette.brand[800],
 	backgroundSecondaryHover: '#ACC9F7', // One-off colour variant generated from palette.brand[800]
 	textTertiary: palette.brand[400],
+	backgroundTertiary: palette.neutral[100],
```

Comment:
can we make this transparent to keep backwards compatibility ?
```suggestion
	backgroundTertiary:  'transparent',
```

Reply:
The white is in Source!

Reply:
Currently if someone is using it on a non white background this is will change it to white as this is the default. Like here in storybook. We could change it in the reader revenue ones to transparent or leave all of them as transparent.
<img width="1226" alt="Screenshot 2024-03-26 at 13 21 36" src="https://github.com/guardian/csnx/assets/40648315/e5305ec9-4e15-4c69-bb31-0f9bce08f44b">

Reply:
As in source it is always the same colour as the background I am inclined to leave this as transparent. Where in source are you referring to?

Reply:
It has a white background in the zeroheight docs, but this may be because the default theme is rendered against a grey background to differentiate it from the rest of the page:
https://theguardian.design/2a1e5182b/p/435225-button

In Figma the background colour is transparent.